core/includes/form.inc

```diff
@@ -3629,17 +3629,23 @@ function form_process_container($element, &$form_state) {
  */
 function theme_container($variables) {
   $element = $variables['element'];
-  // Ensure #attributes is set.
-  $element += array('#attributes' => array());
+  // Ensure #attributes is set, but do not overwrite them if they have already
+  // been set.
+  $attributes = isset($element['#attributes']) ? $element['#attributes'] : array();
+  $element += array('#attributes' => $attributes);
```

Comment:
This part of the change here seems redundant to me. The `+=` already takes care that the array is only initialized, if it doesn't exist yet.

An example to play:
```
$foo = array('attributes' => array('class' => 'foo'));
$foo += array('attributes' => array());
var_dump($foo);
```
^^ The foo class is preserved.

Maybe the [docs](https://www.php.net/manual/en/language.operators.array.php) refresh your memory after your long vacation. :smile:

Reply:
You are absolutely right of course 😅 ...I don't know why this was initially breaking on my local.